rollup/core/classes/RollupLogger.cls

```diff
@@ -40,15 +43,19 @@ public virtual class RollupLogger extends Rollup implements ILogger {
   }
 
   public virtual void log(String logString, Object logObject, LoggingLevel logLevel) {
-    if (this.rollupControl?.IsRollupLoggingEnabled__c == true && logLevel.ordinal() >= this.getLogLevel().ordinal()) {
+    if (this.rollupControl?.IsRollupLoggingEnabled__c == true && logLevel.ordinal() >= this.currentLoggingLevel.ordinal()) {
```

Comment:
this was an embarrassing find while working through the full recalc bulkification with Karl Berlin - the prior version of the code was actually burning quite a bit of CPU time by iterating through the `LoggingLevel` values every time the `log` method was called. Can't wait for `LoggingLevel.valueOf()` (though I'll still cache the value in the constructor, just to be safe)